### ISSUE TITLE
Support units and reference system

### ISSUE THREAD

**Issue by astrojuanlu** [mommermi/callhorizons#16]
Right now the units and the reference system of the output are hardcoded in the query:

https://github.com/mommermi/callhorizons/blob/14fd663/callhorizons/callhorizons.py#L1059-L1061

It would be nice to be able to change them. Similar to #15.

**Comment by mommermi** [comment]
Good point, I never thought of changing these parameters. Let me consider this for `astroquery.jplhorizons` (see #15).
